modules/channel-web/src/views/lite/main.tsx

```diff
@@ -200,6 +201,12 @@ class Web extends React.Component<MainProps> {
     }
   }
 
+  handleClearConversation = async (event: Message) => {
+    if (!this.props.config?.conversationId || Number(this.props.config.conversationId) === Number(event.conversationId)) {
```

Comment:
We now have this condition in like 3 different methods, could you move it to an helper method to avoid duplication? Ex: `isCurrentConversation(event)`